app/editPlan/get.controller.js

```diff
@@ -14,7 +14,10 @@ const editPlan = async ({ path, params: { id, planId }, session: { 'x-auth-token
         getFinalInformation(sentencePlan, stub, objectivesSection.items[0].complete),
       ],
     }
-    res.render(`${__dirname}/index`, { id, token, planSummary })
+
+    const disableStartButton = !planSummary.sections[2].items[0].complete
```

Comment:
I think this has a bit of a code smell about it; It looks like we're accessing the same variable in two ways:
```objectivesSection.items[0].complete```
```planSummary.sections[2].items[0].complete```
Or am I getting confused?
Could we just set it once locally and then put that in the plan summary?

Reply:
It's not the same - objectivesSection is planSummary.sections[1]. I think the way all this works can be made a bit more generic and less code smelly when we've got time though, I agree.

Reply:
Yeah, I was finding it difficult to trace things up the chain and took a punt. I'll leave it up to you to deem it worthy enough to raise a tech debt ticket for.